distribution/sdk/artemis-sandbox/pom.xml

```diff
@@ -55,7 +55,45 @@
             <artifactId>artemis-jms-client</artifactId>
             <version>${artemis.version}</version>
         </dependency>
+        <dependency>
```

Comment:
It is unrelated to fixing the bug. 
However, there is another issue with the sdk examples routes not being able to run due to these missing dependencies.